### PR TITLE
fix(applet): update component render-code when toggle tree-node

### PR DESCRIPTION
**Pull request by CoutinhoTTS**
https://github.com/vuejs/devtools-next/assets/48982306/610294e6-237a-4d05-af74-e9500d4b5810

image
image

### CODE REVIEW

File: packages/applet/src/modules/components/components/RenderCode.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@ function close() {
 </script>
 
 <template>
-  <div class="absolute left-0 top-0 h-full w-full flex flex-col bg-white p2 dark:bg-[#121212]">
+  <div class="absolute left-0 top-0 h-full w-full flex flex-col rounded-br-10px rounded-tr-10px bg-white p2 dark:bg-[#121212]">
```
Comment (webfansplz) [inline packages/applet/src/modules/components/components/RenderCode.vue:16]: 10px -> 2.5

File: packages/applet/src/modules/components/index.vue [hunk 2]
```diff
@@ -37,6 +37,7 @@ const [filtered, toggleFiltered] = useToggle(true)
 const componentTreeLoaded = ref(false)
 const inspectComponentTipVisible = ref(false)
 const componentRenderCode = ref('')
+const showComponentRenderCode = ref(false)
```
Comment (webfansplz) [inline packages/applet/src/modules/components/index.vue:40]: Let's change to `componentRenderCodeVisible`

File: packages/applet/src/modules/components/index.vue [hunk 3]
```diff
@@ -265,7 +270,7 @@ function scrollToActiveTreeNode() {
           </div>
           <RootStateViewer class="no-scrollbar flex-1 select-none overflow-scroll" :data="filteredState" :node-id="activeComponentId" :inspector-id="inspectorId" expanded-state-id="component-state" />
         </div>
-        <ComponentRenderCode v-if="componentRenderCode" :code="componentRenderCode" @close="componentRenderCode = ''" />
+        <ComponentRenderCode v-if="showComponentRenderCode && componentRenderCode" :code="componentRenderCode" @close="componentRenderCode = '';showComponentRenderCode = false" />
```
Comment (webfansplz) [inline packages/applet/src/modules/components/index.vue:273]: Let's create a function to handle it:

```ts
function closeComponentRenderCode() {
  componentRenderCode.value = ''
  componentRenderCodeVisible.value = false
}


```

File: packages/applet/src/modules/components/index.vue (hunk 2 above)
Comment (CoutinhoTTS) [reply packages/applet/src/modules/components/index.vue]: Pushed

**Comment by webfansplz** [comment]
Thanks.

**Pull request merged**